// Exceptions 
function functionError(){
  throw new Error('error is running')
}
module.exports = functionError